lib/fastlane/setup/setup_ios.rb

```diff
@@ -52,6 +45,47 @@ def run
       # rubocop:enable Lint/RescueException
     end
 
+    def default_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: false)
+      detect_installed_tools # after copying the existing files
+      create_app_if_necessary
+      enable_deliver
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: false)
+      show_analytics
+    end
+
+    def manual_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: true)
+      detect_installed_tools # after copying the existing files
+      ask_to_enable_other_tools
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: true)
+      show_analytics
+    end
+
+    def ask_to_enable_other_tools
+      create_app = agree('Would you like to create your app on iTunes Connect and the Developer Portal?', true)
+      if create_app
+        create_app_if_necessary
+      end
+      if create_app and agree("Do you want to setup 'deliver', which is used to upload app screenshots, app metadata and app updates to the App Store? This requires the app to be in the App Store already. (y/n)".yellow, true)
+        enable_deliver
+      end
+
+      if !@tools[:snapshot] and Helper.mac? and agree("Do you want to setup 'snapshot', which will help you to automatically take screenshots of your iOS app in all languages/devices? (y/n)".yellow, true)
```

Comment:
We should remove the `snapshot` code, it's also not there with the automatic mode, right?